Makefile

```diff
@@ -4,7 +4,7 @@ all: gen build
 
 gen: temporalcli/commands.gen.go
 
-temporalcli/commands.gen.go: temporalcli/commandsmd/commands.md
+temporalcli/commands.gen.go: temporalcli/commandsgen/commands.md
```

Comment:
This still references `commands.md`. In a perfect world this would have failed CI with something like:

> No rule to make target 'temporalcli/commandsgen/commands.md', needed by 'temporalcli/commands.gen.go'

But unfortunately this Makefile is an untested/undocumented thing just sitting in the repo

Reply:
oops, thanks for catching this!